internal/scaffold/ansible/go_mod.go

```diff
@@ -41,7 +41,7 @@ const goModTmpl = `module {{ .Repo }}
 go 1.13
 
 require (
-	github.com/operator-framework/operator-sdk master
+	github.com/operator-framework/operator-sdk v0.15.x
```

Comment:
Looks like inadvertent changes to the `go.mod` scaffolds?

Reply:
yikes!  yep, I'll switch those back to master.